### DISCUSSION TITLE
Need option for hide columns

### DISCUSSION THREAD

**Discussion by KonstantinKlepikov** [community/community#15241] [Projects and Issues] [answered]
![image](https://user-images.githubusercontent.com/20581667/164127053-ac966a53-5b31-4aef-b5e5-685f029cb7bf.png)

**Comment by interob** [comment, marked as answer]
I would say this one is a duplicate for #8445:

Hide/unhide could be the same as collapse/restore

**Reply by mesqueeb** [reply]
I would rather have a complete "hide" feature instead of "collapse"
